### ISSUE TITLE
Can I leverage pymobiledevice3 to remote debug iDevice with XCode

### ISSUE THREAD

**Issue by c2uc2u** [doronz88/pymobiledevice3#1265]
**Describe the solution you'd like**
For instance,  I have an iDevice connected to my Windows machine, and another macOS machine on the same network running Xcode. Is it possible to use pymobiledevice3 to remotely access the iDevice on my Windows machine from Xcode for live debugging?

**Comment by c2uc2u** [comment]
I have tried forwarding the usbmuxd plist message from a Windows machine to a macOS machine, but it doesn't seem to work with iOS 17.6 and macOS 15.0.1.
In my test, the macOS Finder can see the remote iOS device, but Xcode cannot...

**Comment by doronz88** [comment]
For xcode to work with your device you'll need to be in the same LAN with the device

**Comment by c2uc2u** [comment]
@doronz88 Thanks, the two machines are in the same LAN and the idevice is connected via USB to one of these machines. Can I debug the idevice from another machine by the XCode with pymobiledevice's help? Could you guid me hwo to setup it with pymobiledevice? 

Here is what I have tried, the following machine are running in the same LAN

- Mac Machine:
Rename “/var/run/usbmuxd” to “/var/run/[usbmuxd.org](http://usbmuxd.org/)”
Running my test program and listening on “/var/run/usbmuxd” instead.
My test program will forward all the plist messages to  “/var/run/[usbmuxd.org](http://usbmuxd.org/)”.
Run socat command 
	“socat tcp-listen:5050,fork unix-connect:/var/run/usbmuxd”

- Windows Machine:
An iOS device connected
pymobiledevice3 usbmux list --usbmux xxx.xxx.xxx.xxxx:5050

In my test, I can see the remote iOS device is attached in the macOS Finder in my Mac Machine, but Xcode cannot see the device.

**Comment by c2uc2u** [comment]
I have tried to set up the trusted tunnel to send the RemoteXPC protocol between my Windows PC (the idevice connected) and Mac PC as well. My iOS device is running iOS 17.6.1, and my macOS is version 15.0.

On my Mac machine, I ran the following commands:

```
sudo python3 -m pymobiledevice3 lockdown start-tunnel --usbmux xxx.xxx.xxx.xxx:5050

sudo python3 -m pymobiledevice3 developer dvt launch com.apple.mobilesafari --rsd fdc3:16b1:5cac::1 52954
```

While I was able to launch the Safari app using this command, it did not resolve the Xcode issue. I still cannot see the device from the Xcode “Device and Simulators” windows in my Mac machine.
